docs/features/lambda_dead_letter_queue.md

```diff
@@ -13,33 +13,28 @@ The [workflow-trigger](../workflows/workflow-triggers) for the startSF queue has
 This queue can then be monitored for failures to initiate a workflow.   Please note that workflow failures will not show up in this queue, only repeated failure to trigger a workflow.
 
 ## Named Lambda Dead Letter Queues
-Cumulus provides the ability to configure a default named [Dead Letter Queue](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html) for lambdas.   This is intended to be utilized for non-workflow lambdas (such as ScheduleSF) to capture lambda failures for further processing.
 
-Adding the following configuration to a lambda in `app/lambdas.yml`:
+Cumulus provides configured [Dead Letter Queues](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html) (`DLQ`) for non-workflow lambdas (such as ScheduleSF) to capture lambda failures for further processing.
 
-```
-namedLambdaDeadLetterQueue: true
-```
-
-will create an SQS Dead Letter Queue named `{lambdaName}DeadLetterQueue` and set the lambda to target it.   The default retention configuration for this queue will be set with the following configurable values:
+These DLQs are setup with the following configuration:
 
+```hcl
+  receive_wait_time_seconds  = 20
+  message_retention_seconds  = 1209600
+  visibility_timeout_seconds = 60
 ```
-DLQDefaultTimeout: 60  ## SQS Message Timer
-DLQDefaultMessageRetentionPeriod: 1209600 ## SQS Message Retention
-```
-
-These values can be overridden in the `app/config.yml` and will apply to all 'named' lambda Dead Letter Queues, subject to [AWS SQS Limits](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-limits.html)
 
-#### Default Lambda Configuration
+### **Default Lambda Configuration**
 
-The following built-in cumulus lambdas have this feature enabled by default to allow handling of process failures:
+The following built-in Cumulus Lambdas are setup with DLQs to allow handling of process failures:
 
 * dbIndexer (Updates Elasticsearch based on DynamoDB events)
 * EmsIngestReport (Daily EMS ingest report generation lambda)
 * JobsLambda (writes logs outputs to Elasticsearch)
 * log2elasticsearch (Lambda that exports logs into Elasticsearch)
-* ScheduleSF (The SF Scheduler lambda that places messages on the start SF queue, see [Workflow Triggers](../workflows/workflow-triggers.md))
-* sns2elasticsearch (API Lambda that takes a payload from a workflow and indexes it into Elasticsearch)
+* ScheduleSF (the SF Scheduler lambda that places messages on the start SF queue, see [Workflow Triggers](../workflows/workflow-triggers.md))
```

Comment:
Can we say "the queue that is used to start workflows" instead of "start SF queue"? I don't want to tie ourselves to a specific queue name in the docs that we may have to update later. It is more wordy though, though "start SF queue" probably needs a bit more context either way

Reply:
I don't know that I mind wordy in this context.     That seems like an acceptable abstraction given we're linking to the triggers doc. 

b6824828c